kitsune/products/views.py

```diff
@@ -21,6 +23,14 @@ def product_list(request, template):
         'products': products})
 
 
+def is_localized(slug, request_language_code):
```

Comment:
Having  ``parent_slug`` and ``locale`` this two parameters makes is good readability